src/Resources/symfony/manala.yml

```diff
@@ -36,7 +36,14 @@ packages:
         required:   false
         enabled:    false
 
-script:
-    post_provision:
+make_targets:
+    install:
         - composer install --verbose
         - bin/console doctrine:database:create --if-not-exists
+    setup:
+        - if [ -d  "./var/cache" ]; then rm -rf ./var/cache && mkdir ./var/cache; fi;
```

Comment:
This is a workaround mandatory until @nervo automagically dumps all the files from these directories into a temp dir then redump them after provisioning (I have not the heart to write such in the makefile, this is already so ugly)